recipes/cppcheck/all/conanfile.py

```diff
@@ -44,7 +44,7 @@ def generate(self):
         tc.variables["HAVE_RULES"] = self.options.have_rules
         tc.variables["USE_MATCHCOMPILER"] = "Auto"
         tc.variables["ENABLE_OSS_FUZZ"] = False
-        tc.variables["FILESDIR"] = os.path.join(self.package_folder, "res").replace('\\', '/')
+        tc.variables["FILESDIR"] = os.path.join(self.package_folder, "bin").replace('\\', '/')
```

Comment:
maybe extend the test to check that nobody will break it again?

Reply:
unfortunately I don't think we can make a breaking test, because cppcheck stores the absolute path to this FILESDIR folder [at build time](https://github.com/danmar/cppcheck/blob/41849b1a78dc572217204549c5d9532935a75d19/lib/library.cpp#L105)  and uses it as a fallback if the .cfg files cannot be found in the folder next to the cppcheck executable. It means we can only break by downloading an existing package binary into a package folder different from the one used during build :/